README.md

```diff
@@ -1,11 +1,7 @@
-# Avail-da-module
+# Cada (Cosmos Avail DA module)
 
-### Posting cosmos sdk based applications data to Avail DA
-
-We’ve created a module that allows Cosmos SDK-based blockchain applications to post data to Avail DA. By integrating this module into your application, you can easily send your data to the Avail light client, where it will be validated and then posted to the DA.
-
-You can integrate this module into both Cosmos SDK-based applications and those generated with Spawn, which also uses the Cosmos SDK.
+CADA is a module designed to connect Cosmos sovereign chains with the Avail network, making it easier for any Cosmos chain or rollapp to use Avail as their Data Availability (DA) layer. With CADA, developers can improve the scalability and security of their decentralized applications within the Cosmos ecosystem. It enables better data handling and availability, allowing Cosmos-based chains to tap into the strengths of Avail and build a more connected and resilient blockchain network.
```

Comment:
CADA is a module designed to connect Cosmos sovereign chains with the Avail network, facilitating the use of Avail as a Data Availability (DA) layer for any Cosmos chain or rollapp. By integrating CADA, developers can enhance the scalability and security of their decentralized applications within the Cosmos ecosystem. It improves data handling and availability, enabling Cosmos-based chains to leverage Avail’s strengths and build a more interconnected and resilient blockchain network.